pkg/runcfanotify/runcfanotify.go

```diff
@@ -606,6 +639,12 @@ func (n *RuncNotifier) watchRuncIterate() (bool, error) {
 
 func (n *RuncNotifier) Close() {
 	n.closed = true
+	n.awakeWatchContainersTermination()
+	for _, fd := range n.pipeFds {
+		if fd != -1 {
+			unix.Close(fd)
```

Comment:
Should we make `RuncNotifier.Close()` safe if it is called several times?

Reply:
I don't think we should care about that case in all places. I'd leave this up to the caller.